File: copy_model/FamilyMember.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;
use Illuminate\Database\Eloquent\Relations\BelongsTo;

class FamilyMember extends Model
{
    use HasFactory;

    protected $fillable = [
        'user_id',
        'no',
        'name',
        'role',
        'phone',
        'rsvp_status',
        'rsvp_updated_by_name',
        'rsvp_updated_at',
    ];

    protected $casts = [
        'rsvp_updated_at' => 'datetime',
    ];

    public static array $rsvpOptions = [
        'menunggu'     => 'Menunggu',
        'hadir'        => 'Hadir',
        'tidak_hadir'  => 'Tidak Hadir',
    ];

    public function user(): BelongsTo
    {
        return $this->belongsTo(User::class);
    }
}

File: ios-app/WeddingApp/Sources/Models/FamilyMember.swift
import Foundation

struct FamilyMember: Codable, Identifiable, Hashable {
    let id: Int
    var no: Int?
    var name: String
    var role: String?
    var phone: String?
    var rsvpStatus: String
    var rsvpUpdatedByName: String?
    var rsvpUpdatedAt: String?

    var subtitleLine: String {
        guard let role, !role.isEmpty else {
            return L10n.Guest.familyMemberFallback
        }
        return role
    }
}
